### ISSUE TITLE
Lag when editing Markdown

### ISSUE THREAD

**Issue by ghost** [HappenApps/Quiver#555]
With a really long notes in markdown, Quiver lags so much that it is becoming unusable. For example: I have some notes on Rest that consist of several quite long passages of text. Due to lag I can't edit it. This occurs whenever I want to change the markdown tags, for example from #Heading to ### Heading.

**Comment by ylian** [comment]
Is the whole note in a single Markdown cell?

**Comment by ghost** [comment]
Yes, that's the case.

**Comment by ylian** [comment]
Right, Quiver is not yet optimized for that use case. It will help with the performance if you break the note into a few cells. Optimization for this use case in on the roadmap.
